aiotg/chat.py

```diff
@@ -8,6 +9,22 @@ class Chat:
     """
     Wrapper for telegram chats, passed to most callbacks
     """
+    async def response(self, regex=r'.+', unexpected=None):
+        """
+        Await a message if not match with regex send unexpected message.
+
+        :param str regex: Regex match with message sender
+        :param str unexpected: Message if not match
+        :return Response
+        """
+        message = await self.bot._conversation[self.id].get()
```

Comment:
First of all, this is needs to be wrapped in try catch or dict get :)

But actually, I suggest to init queue here:
```
conversation = self.bot._conversations.get(self.id)
if not conversation:
   self.bot._conversations.get[self.id] = asyncio.Queue(maxsize=3)
message = await conversation.get()
```

Reply:
And what do you think
```python
self.bot._conversations[self.id] = asyncio.Queue()
message = await self.bot._conversations[self.id].get()
del self.bot._conversations[self.id]
```
so unnecessary conversations are not held